velox/exec/StreamingAggregation.cpp

```diff
@@ -61,8 +61,25 @@ void StreamingAggregation::initialize() {
   aggregates_ = toAggregateInfo(
       *aggregationNode_, *operatorCtx_, numKeys, expressionEvaluator, true);
 
+  // Setup SortedAggregations.
```

Comment:
nit: perhaps, extract this logic into a helper method

Reply:
Fixed, added AggregationInfo::extractMaskChannels, SortedAggregation::extractSortedAggregations method, used by both GroupingSet and StreamingAggregation.